tasks/src/app/custom-button/custom-button.component.ts

```diff
@@ -7,15 +7,19 @@ import { Component, OnInit } from '@angular/core';
 })
 export class CustomButtonComponent implements OnInit {
 
-	public numOfClicks: number = 0;
+	public isInputEnabled: boolean = false;
+	public inputText: string = '';
 
 	public constructor() { }
 
 	public ngOnInit(): void {
 	}
 
-	public increment(): void {
-		this.numOfClicks++;
+	public toggleInput(): void {
+		this.isInputEnabled = !this.isInputEnabled;
 	}
 
-}
+	public updateGreet(event: any): void {
+		this.inputText += event.target.value[event.target.value.length - 1];
```

Comment:
```suggestion
		this.inputText = event.target.value;
```
you can copy the value from the input component and store it in this.inputText